src/renderers/dom/shared/__tests__/ReactDOMServerIntegration-test.js

```diff
@@ -227,21 +251,28 @@ function itClientRenders(desc, testFn) {
   });
 }
 
-function itThrows(desc, testFn) {
+function itThrows(desc, testFn, partialMessage) {
```

Comment:
Adding this to every error. Helped uncover cases where we crashed instead of throwing something useful.